client/src/api/connectJSON.js

```diff
@@ -42,3 +42,14 @@ export async function deleteRecipeFromWeek(Id) {
     method: "DELETE",
   });
 }
+
+export async function updateWeekByDate(date, Id) {
+  const saveDate = { date: date };
```

Comment:
```suggestion
  const saveDate = { date };
```

If key and value are identical, you can use this short form.

Reply:
oh thanks, didn't know that! 😍